plugins/infinispan100/src/main/java/org/radargun/service/InfinispanRestAPI.java

```diff
@@ -23,46 +22,74 @@
  */
 public class InfinispanRestAPI {
 
+   protected final Log log = LogFactory.getLog(getClass());
+
    private ObjectMapper mapper;
-   private HttpClient httpClient;
-   private HttpResponse<String> response;
    private Integer defaultPort;
    private static final String CACHE_MANAGER_RESOURCE = "v2/cache-managers/clustered";
    private String serverIp;
-   private final Log log = LogFactory.getLog(getClass());
 
    public InfinispanRestAPI(Integer defaultPort) {
-      mapper = new ObjectMapper();
-      mapper.disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
-      mapper.setPropertyNamingStrategy(PropertyNamingStrategy.SNAKE_CASE);
-      httpClient = HttpClient.newBuilder().version(HttpClient.Version.HTTP_1_1).build();
```

Comment:
@gustavolira sorry about removing this code, but I didn't think about the consequences of relying on JDK 11 library code when it went into the codebase.

Reply:
Yeah. I am sorry as well.

Reply:
np